backend/src/packages/chaiNNer_standard/image/create_images/text_as_image.py

```diff
@@ -0,0 +1,271 @@
+    },
+    TextAsImageFont.OPEN_SANS_REGULAR: {
+        "label": "Open Sans",
+        "path": "OpenSans/OpenSans-Regular.ttf",
+    },
+    TextAsImageFont.OPEN_SANS_BOLD: {
+        "label": "Open Sans Bold",
+        "path": "OpenSans/OpenSans-Bold.ttf",
+    },
+    TextAsImageFont.OPEN_SANS_ITALIC: {
+        "label": "Open Sans Italic",
+        "path": "OpenSans/OpenSans-Italic.ttf",
+    },
+    TextAsImageFont.OPEN_SANS_BOLD_ITALIC: {
+        "label": "Open Sans Bold Italic",
+        "path": "OpenSans/OpenSans-BoldItalic.ttf",
+    },
+}
+
+TEXT_AS_IMAGE_FONT_LABELS = {
+    key: TEXT_AS_IMAGE_FONTS[key]["label"] for key in TextAsImageFont
+}
+
+
+class TextAsImageAlignment(Enum):
+    LEFT = "left"
+    CENTERED = "center"
+    RIGHT = "right"
+
+
+TEXT_AS_IMAGE_ALIGNMENT_LABELS = {
+    TextAsImageAlignment.LEFT: "Left",
+    TextAsImageAlignment.CENTERED: "Centered",
+    TextAsImageAlignment.RIGHT: "Right",
+}
+
+
+class TextAsImagePosition(Enum):
+    TOP_LEFT = "top_left"
+    TOP_CENTERED = "top_centered"
+    TOP_RIGHT = "top_right"
+    CENTERED_LEFT = "centered_left"
+    CENTERED = "centered"
+    CENTERED_RIGHT = "centered_right"
+    BOTTOM_LEFT = "bottom_left"
+    BOTTOM_CENTERED = "bottom_centered"
+    BOTTOM_RIGHT = "bottom_right"
+
+
+TEXT_AS_IMAGE_POSITION_LABELS = {
+    TextAsImagePosition.TOP_LEFT: "Top left",
+    TextAsImagePosition.TOP_CENTERED: "Top centered",
+    TextAsImagePosition.TOP_RIGHT: "Top right",
+    TextAsImagePosition.CENTERED_LEFT: "Centered left",
+    TextAsImagePosition.CENTERED: "Centered",
+    TextAsImagePosition.CENTERED_RIGHT: "Centered right",
+    TextAsImagePosition.BOTTOM_LEFT: "Bottom left",
+    TextAsImagePosition.BOTTOM_CENTERED: "Bottom centered",
+    TextAsImagePosition.BOTTOM_RIGHT: "Bottom right",
+}
+
+TEXT_AS_IMAGE_X_Y_REF_FACTORS = {
+    TextAsImagePosition.TOP_LEFT: {"x": np.array([0, 0.5]), "y": np.array([0, 0.5])},
+    TextAsImagePosition.TOP_CENTERED: {
+        "x": np.array([0.5, 0]),
+        "y": np.array([0, 0.5]),
+    },
+    TextAsImagePosition.TOP_RIGHT: {"x": np.array([1, -0.5]), "y": np.array([0, 0.5])},
+    TextAsImagePosition.CENTERED_LEFT: {
+        "x": np.array([0, 0.5]),
+        "y": np.array([0.5, 0]),
+    },
+    TextAsImagePosition.CENTERED: {"x": np.array([0.5, 0]), "y": np.array([0.5, 0])},
+    TextAsImagePosition.CENTERED_RIGHT: {
+        "x": np.array([1, -0.5]),
+        "y": np.array([0.5, 0]),
+    },
+    TextAsImagePosition.BOTTOM_LEFT: {
+        "x": np.array([0, 0.5]),
+        "y": np.array([1, -0.5]),
+    },
+    TextAsImagePosition.BOTTOM_CENTERED: {
+        "x": np.array([0.5, 0]),
+        "y": np.array([1, -0.5]),
+    },
+    TextAsImagePosition.BOTTOM_RIGHT: {
+        "x": np.array([1, -0.5]),
+        "y": np.array([1, -0.5]),
+    },
+}
+
+
+@create_images_group.register(
+    schema_id="chainner:image:text_as_image",
+    name="Text As Image",
+    description="Convert a text to an image.",
```

Comment:
Maybe something more like this?
```suggestion
    description="Create an image using any text and a chosen font.",
```